plugin/applescript/update.scpt

```diff
@@ -3,11 +3,12 @@ tell application "System Events"
 end tell
 
 on update_xmark()
-    tell application "{{ app }}"
+    tell application "{{ app_loc }}"
```

Comment:
`close.scpt` (which is called on `:Xmark!`) also has the code. Shouldn't we fix it as well?